javascript/javascript2/week3/gstyle.css

```diff
@@ -0,0 +1,55 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap");
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  min-height: 100vh;
+  font-family: "Times New Roman", Times, serif;
+  padding: 2rem 3%;
+}
+
+h1 {
+  font-size: 2.5rem;
+  font-weight: bold;
+}
+
+button {
+  background-color: green;
+  color: #fff;
+  font-size: 1rem;
+  padding: 6px;
```

Comment:
Since you are already using 'rem' for a padding (of the body), I would highly recommend using it here as well and elsewhere, to be consistent. But of course this is totally optional and a more advanced topic :)